tools/Messages

```diff
@@ -325,42 +330,50 @@ do
             echo "Uknown option \"$opt\"" && Usage && exit 5
         ;;
         *)
-            if [ $# -ne 1 ]
-            then
-                echo "Bad syntax. The text should be the last argument."
-                echo "Stopped near \"$@\"."
-                echo "Check the quotes around the arguments?"
-                exit 3
-            fi
-            if [ "x$MESSAGE_TEXT" = "x" ]
-            then
-                MESSAGE_TEXT="$@"
-                break # exit the while cycle
-            else
-                echo "Bad syntax. The text must be specified only once."
-                exit 3
-            fi
+	    if [ $MESSAGE_TYPE == "config" ]; then
+		MESSAGE_FILE="$@"
+		MESSAGE_TEXT="load"
+	    else
+                if [ $# -ne 1 ]
+                then
+                    echo "Bad syntax. The text should be the last argument."
+                    echo "Stopped near \"$@\"."
+                    echo "Check the quotes around the arguments?"
+                    exit 3
+                fi
+                if [ "x$MESSAGE_TEXT" = "x" ]
+                then
+                    MESSAGE_TEXT="$@"
+                    break # exit the while cycle
+                else
+                    echo "Bad syntax. The text must be specified only once."
+                    exit 3
+                fi
+	    fi
         ;;
     esac
     shift
 done
 
 
-[ -z "$MESSAGE_TEXT" ] && echo "The text of the message must be specified" && Usage && exit 3
-[ -z "$MESSAGE_EXPIRE" ] && echo "The expire must be set" && Usage && exit 5
+if [ $MESSAGE_TYPE != "config" ]; then
+    [ -z "$MESSAGE_TEXT" ] && echo "The text of the message must be specified" && Usage && exit 3
+    [ -z "$MESSAGE_EXPIRE" ] && echo "The expire must be set" && Usage && exit 5
 
-if [ -z "$MESSAGE_EXPIRE_TIMESTAMP" ]
-then
-    MESSAGE_EXPIRE_TIMESTAMP=`"$DATE" --date="$MESSAGE_EXPIRE" +"%s"`
-    if [ $? -ne 0 ]
+    if [ -z "$MESSAGE_EXPIRE_TIMESTAMP" ]
```

Comment:
`./tools/Messages -v -n config /var/opengrok/etc/configuration.xml` results in:

`
/usr/jdk/latest/bin/java -Xmx2048m -classpath :/export/home/vkotal/opengrok-git-vladak/tools/../dist/opengrok.jar org.opensolaris.opengrok.configuration.messages.Messages -m config -c info -g load -s localhost -p 2424 -f /var/opengrok/etc/configuration.xml --
`